kong/router/transform.lua

```diff
@@ -546,6 +570,14 @@ local function get_priority(route)
 
   local match_weight = 0  -- 0x0ULL
```

Comment:
`match_weight` only have 3 bits, and we are using all of them with this change. We should add a comment here saying this number can not be more than `7`, otherwise bad things might happen.

Reply:
done.